scripts/darwin.sh

```diff
@@ -1,35 +1,38 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
A lot of the shellcheck output was based on using POSIX sh instead of bash (as this script was using bash-style features), but you've now switched to bash here anyway!

So IMHO you should either leave it as `/bin/sh` and fix up all the shellcheck errors (and you need to re-run it after every change before committing) to make the script sh-compliant, OR you should revert all the changes you've made here, switch to bash, and then re-run shellcheck (which will then presumably then operate in bash-mode, and not point out all the POSIX sh errors). Hope that makes sense ;-)